addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/internal/net/Slf4jLoggingFilter.java

```diff
@@ -0,0 +1,417 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.neeo.internal.net;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.concurrent.atomic.AtomicLong;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientRequestFilter;
+import javax.ws.rs.client.ClientResponseContext;
+import javax.ws.rs.client.ClientResponseFilter;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.container.PreMatching;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.WriterInterceptor;
+import javax.ws.rs.ext.WriterInterceptorContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Logging filter to use for Slf4j
+ *
+ * @author Tim Roberts - Initial Contribution
+ */
+@PreMatching
+public class Slf4jLoggingFilter implements ContainerRequestFilter, ClientRequestFilter, ContainerResponseFilter,
+        ClientResponseFilter, WriterInterceptor {
```

Comment:
Actually it's not - it's a modified version of something I found on stack trace (which I believe is MIT but who knows if the author borrowed from the class you are talking about) - not sure what openHAB license requires but let me know

Reply:
Those logging filters are really for development purposes actually - I can eliminate them (or gitignore them) instead and we don't need to worry about licenses then

Reply:
I removed the class and added a much more simplified version that does only what I really need in a debug concept (in case I need it in the logs for someone)